static/css/main.css

```diff
@@ -390,9 +537,14 @@ body {
 }
 
 @media (max-width: 750px) {
+  .x.axis {
+    font-size: 2.5vmin !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)